libparsec/crates/client/src/certificates_ops/poll.rs

```diff
@@ -4,6 +4,8 @@ use libparsec_client_connection::ConnectionError;
 use libparsec_protocol::authenticated_cmds;
 use libparsec_types::prelude::*;
 
+use crate::certificates_ops::store::CertificatesStoreReadExt;
+
```

Comment:
As a general rule of thumb in the codebase we have `crate` and `super` uses not separated by a new line (newline separates 3rd party then 3) parsec crates, then our own crate), and crate before super (given crate is more broad than super)

Reply:
You have to choose between the separation or super coming first :sweat_smile: